src/main/resources/templates/layout.html

```diff
@@ -11,14 +11,14 @@
 
     <title>Amazin</title>
     <!-- Custom fonts for this template-->
-    <link rel="stylesheet" type="text/css"
-          href="/webjars/font-awesome/5.7.2/css/all.min.css" />
+    <link rel="stylesheet" type="text/css" href="/webjars/font-awesome/5.7.2/css/all.min.css" />
 
-    <link href="https://fonts.googleapis.com/css?family=Nunito:200,200i,300,300i,400,400i,600,600i,700,700i,800,800i,900,900i" rel="stylesheet">
+    <link
+        href="https://fonts.googleapis.com/css?family=Nunito:200,200i,300,300i,400,400i,600,600i,700,700i,800,800i,900,900i"
+        rel="stylesheet">
 
     <!-- Custom styles for this template-->
-    <link rel="stylesheet"
-          href="/webjars/startbootstrap-sb-admin-2/4.0.2/css/sb-admin-2.min.css" />
+    <link rel="stylesheet" href="/webjars/startbootstrap-sb-admin-2/4.0.2/css/sb-admin-2.min.css" />
```

Comment:
lots of lines modified from the linter it seems 🤔

Reply:
It's hard to tell if it was only linting/indentation changes in this file or if anything was added to it

Reply:
the latest commit highlights exaclty what was changed by me that is not linting